data/brands/amenity/cinema.json

```diff
@@ -59,6 +59,31 @@
         "brand:wikipedia": "en:AMC Theatres",
         "name": "AMC"
       }
+     },
+     {	
+      "displayName": "Irish Multiplex Cinemas",
+      "id": "",
+      "locationSet": {"include": ["ie"]},
+      "matchNames": ["imc"],
+      "tags": {
+        "amenity": "cinema",
+        "brand": "Irish Multiplex Cinemas",
```

Comment:
Should we exchange `brand` and `name`?

Reply:
For matchNames?

Reply:
I means, will Ireland community prefer the abbreviation as `name` tag and let the full name as `brand` ? This maybe a bit strange. (or you can use `"short_name": "IMC"`)

Reply:
It is a shorter name yes, but they are always be called "IMC *location*". On the buildings they usually also just have the logo which is just IMC and in some instances it would be the logo followed by a location underneath. So I would say the abbreviation is preferred.